xml/ceph_prompts.xml

```diff
@@ -117,23 +107,33 @@
 &prompt.cephuser.mon;
 </screen>
   </tip>
- </sect1>
- <sect1>
-  <title>General Linux Commands</title>
+  <sect3>
+   <title>Running &ceph; Commands inside Containers</title>
+   <para>
+    Several executables&mdash;such as <command>ceph-volume</command>&mdash;
+    reside inside a running container. You cannot run them directly from the
+    underlying operating system, but via the <command>cephadm</command> module,
+    for example:
+   </para>
+<screen>&prompt.cephuser;cephadm ceph-volume simple scan</screen>
```

Comment:
I'm not sure `cephadm ceph-volume`, which is a special case, is a good example. Other executables that are available only in the container image would be run using `cephadm shell`, which uses a different syntax.

Also, I don't think the wording "reside inside a running container" is correct. When the user issues a command like `cephadm ceph-volume` or `cephadm shell`, cephadm executes podman, which spawns a container from the ceph base container image. The commands actually reside inside this container image, not inside "a running container". The container doesn't actually run long enough to say something "resides" in it. But maybe I'm splitting hairs here.

Maybe @sebastian-philipp could suggest a better wording?